project/Dependencies.scala

```diff
@@ -22,7 +22,7 @@ object Dependencies {
 
   lazy val akkaTest = Seq(akkaPersistenceTestkit).map(_ % akkaVersion)
 
-  lazy val pekkoVersion = "1.0.3"
+  lazy val pekkoVersion = "1.1.0"
```

Comment:
Shall we wait for this version bump due to the kamon issue?

Reply:
hmm nevermind, this is a provided dependency,

Reply:
yep indeed